pkg/service/types.go

```diff
@@ -12,29 +12,15 @@ type OperatorExample struct {
 	Spec              map[string]interface{} `json:"spec,omitempty"`
 }
 
-// ServiceInfo holds all important information about one service
-type Service struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              ServiceSpec   `json:"spec,omitempty"`
-	Status            ServiceStatus `json:"status,omitempty"`
-}
-
-// ServiceSpec ...
-type ServiceSpec struct {
-	Type string `json:"type,omitempty"`
-	Plan string `json:"plan,omitempty"`
-}
-
-// ServiceStatus ...
-type ServiceStatus struct {
-	Status string `json:"status,omitempty"`
-}
-
-type ServiceList struct {
-	metav1.TypeMeta `json:",inline"`
-	metav1.ListMeta `json:"metadata,omitempty"`
-	Items           []Service `json:"items"`
+// ServiceClass holds the information regarding a service catalog service class
```

Comment:
I think this comes from an old version. I cannot see how it is used

Reply:
Do we want to remove them? they are operator related correct

Reply:
And yea il look at the files

Reply:
I'm talking about `ServiceClass`. It is not used anywhere on the code

Reply:
About the files I found there are functions that are not specific to `service catalog`. I will see what I can do

Reply:
Sorry I was refering to the functiosn you listed, ServiceClass yea, it needs to go